src/services/file/types.ts

```diff
@@ -1 +1,10 @@
+import { UnionOfConst } from '@graasp/sdk';
+
 export type BuildFilePathFunction = (itemId: string, filename: string) => string;
+
+export const FileStorage = {
+  Local: 'local',
+  S3: 's3',
+};
+Object.freeze(FileStorage);
```

Comment:
Interesting. I read that the difference between `as const` and `Object.freeze()` is that `as const` is compile time and `Object.freeze` is runtime. Usually I use:
```ts
export const FileStorage = {
  Local: 'local',
  S3: 's3',
} as const;
```
But you are right that `Object.freeze` also ensures that at runtime nothing modifies the object, whereas `as const` does not give such guaranties. But it comes with some overhead.